state-chain/pallets/cf-emissions/src/lib.rs

```diff
@@ -171,18 +182,7 @@ pub mod pallet {
 			T::RewardsDistribution::distribute();
 			if Self::should_update_supply_at(current_block) {
 				if T::SafeMode::get().emissions_sync_enabled {
-					let flip_to_burn = T::FlipToBurn::take_flip_to_burn();
-					if flip_to_burn > Zero::zero() {
-						T::EgressHandler::schedule_egress(
-							Asset::Flip,
-							flip_to_burn,
-							ForeignChainAddress::Eth(
-								T::EthEnvironment::state_chain_gateway_address(),
-							),
-							None,
-						);
-						T::Issuance::burn(flip_to_burn.into());
-					}
+					Self::burn_flip_network_fee();
```

Comment:
I move this to function to make it easier to test independently of inflation (which is hard to predict).